pkg/genericapiserver/storage_factory.go

```diff
@@ -104,12 +104,21 @@ const AllResources = "*"
 // specialDefaultResourcePrefixes are prefixes compiled into Kubernetes.
 // TODO: move out of this package, it is not generic
 var specialDefaultResourcePrefixes = map[schema.GroupResource]string{
-	schema.GroupResource{Group: "", Resource: "replicationControllers"}: "controllers",
-	schema.GroupResource{Group: "", Resource: "replicationcontrollers"}: "controllers",
-	schema.GroupResource{Group: "", Resource: "endpoints"}:              "services/endpoints",
-	schema.GroupResource{Group: "", Resource: "nodes"}:                  "minions",
-	schema.GroupResource{Group: "", Resource: "services"}:               "services/specs",
-	schema.GroupResource{Group: "extensions", Resource: "ingresses"}:    "ingress",
+	schema.GroupResource{Group: "", Resource: "replicationControllers"}:        "controllers",
+	schema.GroupResource{Group: "", Resource: "replicationcontrollers"}:        "controllers",
+	schema.GroupResource{Group: "", Resource: "endpoints"}:                     "services/endpoints",
+	schema.GroupResource{Group: "", Resource: "nodes"}:                         "minions",
+	schema.GroupResource{Group: "", Resource: "services"}:                      "services/specs",
+	schema.GroupResource{Group: "", Resource: "service"}:                       "services/specs",
+	schema.GroupResource{Group: "", Resource: "serviceaccounts"}:               "serviceAccounts",
+	schema.GroupResource{Group: "", Resource: "resourcequotas"}:                "resourceQuotas",
+	schema.GroupResource{Group: "", Resource: "limitranges"}:                   "limitRanges",
+	schema.GroupResource{Group: "", Resource: "podtemplates"}:                  "podTemplates",
+	schema.GroupResource{Group: "", Resource: "persistentvolumes"}:             "persistentVolumes",
+	schema.GroupResource{Group: "", Resource: "persistentvolumeclaims"}:        "persistentVolumeClaims",
+	schema.GroupResource{Group: "", Resource: "configmaps"}:                    "configMaps",
```

Comment:
remove all the mixed-case mappings you added... those were correctly being lowercased before